packages/flutter_driver/lib/src/driver/vmservice_driver.dart

```diff
@@ -195,19 +195,26 @@ class VMServiceFlutterDriver extends FlutterDriver {
       });
     }
 
+    // The Dart VM may be running with --pause-isolates-on-start.
+    // Set a listener to unpause new isolates as they are ready to run,
+    // otherwise they'll hang indefinitely.
+    client.onIsolateRunnable.listen((VMIsolateRef isolateRef) async {
+      resumeLeniently(await isolateRef.load());
```

Comment:
Is there a reason we're not just setting the outter `isoalte` to `isolateRef.load()` here?

Should we be asserting `isolate == isolateRef.load()` somehow?

Reply:
Ahh I see, this could be other isolates the application uses that also started paused, right?